app/controllers/annotation_controller.js

```diff
@@ -0,0 +1,168 @@
+import Annotation from '../models/annotation';
+
+export const createAnnotation = (req, res) => {
+  if (req.isAuthenticated()) {
```

Comment:
Rather than having this `req.isAuthenticated()` check in a bunch of places, you could include the something like the following in `server.js` to check that the user is authenticated before they access any part of the app:
```
function(req, res, next) {
  if(req.isAuthenticated()) {
    next();
  } else {
    res.redirect('/login');
  }
}
```
important to note that this would have to be defined before the `app.use('/', router);` line in `server.js`